### ISSUE TITLE
Standardize descriptions of HubMAP/SenNet dataset types via UBKG

### ISSUE THREAD

**Issue by AlanSimmons** [x-atlas-consortia/ubkg-etl#138]
# Statement of problem
There are cases in which a dataset type will have multiple alternative descriptions associated with it--i.e., strings with different spellings, different uses of case, etc. This results in dataset types that have multiple facets in faceted search in the UI. Following is an example for seqFISH datasets:

![image](https://github.com/x-atlas-consortia/ubkg-etl/assets/10928372/0565cf3d-25fb-4308-b94d-e9f069c4f334)

[This spreadsheet](https://docs.google.com/spreadsheets/d/1PYe1NYZ8RXJ1_sp6C7Rjtock09x7mjppERpzIZU6S58/edit?usp=sharing) summarizes a gap analysis that identifies descriptions that need to be standardized. There are at least two Slack threads in the multi-assay private channel that discuss the issue, too.

# Proposed solution

1. Update the UBKG data to model the new set of dataset types, along with agreed standard descriptions.
2. Update endpoints in the UBKG API related to datasets to reflect changes in the UBKG data

**Comment by AlanSimmons** [comment]
# Reason for problem: Analysis 
tl;dr

The deployment of "soft assays" and the assay classifier introduced multiple sources of truth for dataset descriptions.

There are currently multiple sources of description for dataset types, including:

- the Entity API
- the [assay classifier](https://github.com/hubmapconsortium/ingest-api/blob/main/src/routes/assayclassifier/testing_rule_chain.json) (aka Rules Engine)

Before the deployment of "soft assays", dataset metadata specific to the UI (including description) were managed in [assay_types.yaml](https://github.com/hubmapconsortium/search-api/blob/main/src/search-schema/data/definitions/enums/assay_types.yaml) in the search-api repo. The YAML file assumed a static "data type" that corresponded to a key for processing workflows (also known as "assay type").

The HuBMAP/SenNet UBKG modeled and extended assay_types.yaml, as can be seen in the current [datasets](https://ontology.api.hubmapconsortium.org/datasets?application_context=HUBMAP) endpoint.

With the deployment of the Rules Engine, the UBKG stopped being a reliable source of truth for dataset metadata. Metadata such as descriptions for dataset types became a product of the Rules Engine.

**Comment by AlanSimmons** [comment]
# Plan

I believe that this is mainly a task of adding new content to the existing UBKG data model, as opposed to enhancing the existing data model.

1. Compare current UBKG content against the gap analysis spreadsheet, with reference to the Rules Engine.
2. Update UBKG content.
3. Enhance UBKG API endpoints.

**Comment by AlanSimmons** [comment]
https://docs.google.com/spreadsheets/d/1PYe1NYZ8RXJ1_sp6C7Rjtock09x7mjppERpzIZU6S58/edit?usp=sharing

Analysis and plan
